src/shogun/features/StringFeatures.h

```diff
@@ -20,6 +20,7 @@
 #include <shogun/features/Features.h>
 #include <shogun/features/Alphabet.h>
 #include <shogun/lib/SGString.h>
+#include <shogun/lib/SGStringList.h>
```

Comment:
was there a compile error before ?

Reply:
There was in my case. I use gcc version 7.3.0.

Reply:
I removed this. The example works fine when I use SGStringList type to store the result of get_string_list (before it was get_features)